Vic2ToHoI4/Source/HOI4World/Map/Region.h

```diff
@@ -0,0 +1,39 @@
+#ifndef REGION_H
+#define REGION_H
+
+
+
+#include "newParser.h"
+
+
+
+namespace HoI4
+{
+	
+class Region: commonItems::parser
+{
+	public:
+		explicit Region(std::istream& theStream);
+
+		[[nodiscard]] auto getID() const { return ID; }
+		[[nodiscard]] auto getName() const { return name; }
```

Comment:
const auto& for non-primitives?

Reply:
In this case changed it to use move mechanics